cmd/metrics-router.go

```diff
@@ -0,0 +1,33 @@
+/*
+ * Minio Cloud Storage, (C) 2018 Minio, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package cmd
+
+import (
+	router "github.com/gorilla/mux"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+const (
+	prometheusMetricsPath = "/metrics"
```

Comment:
To avoid shooting ourselves in the foot in future we can move this to `/prometheus/metrics`  instead - provides room for future work.

Reply:
prometheusMetricsPath changed to /prometheus/metrics